### ISSUE TITLE
Another issue with War

### ISSUE THREAD

**Issue by opengrip** [Nik-Potokar/XIVComboWayTooAkiiified#25]
Noticed another issue with war. When using Infuriate to gain Nascent Chaos your main combo button/fell cleave isn't being replaced by Inner Chaos. I assume it is supposed to change since with the AOE button (overpower/tempest) either will swap into Chaotic Cyclone as soon as you pop infuriate. Its also a bit confusing since if your main combo is already on a fell cleave for not overcapping... so say you were at 50 and used infuriate to hit 100 your main combo switches to a fell cleave instead of a Inner Chaos. Which shouldn't even be on your bar since fell cleave during Nascent Chaos should automatically switch to Inner Chaos. Hope I explained this well enough.

**Comment by Codemned** [comment]
This is intentional so you can pre buff with Infuriate before pulling.

**Comment by opengrip** [comment]
Instead of doing it this way could you make it so it doesn't change until after ST buff?

**Comment by Codemned** [comment]
That's not a bad idea I'll look into.

**Comment by opengrip** [comment]
Awesome this would make it 99% smoother since the only time you really would hold it is during the opener :)

**Comment by Codemned** [comment]
Yeah I know thats why it is like it is but it didn't even occur to me that I could make it conditional instead, I'll try to get it implemented soon.

**Comment by opengrip** [comment]
Very awesome the upheaval changes are working great as well! So far im loving this plugin. If I find any other issues while playing around with other jobs ill make sure to report as well.

**Comment by Codemned** [comment]
It's done and tested it'll be release sometime tomorrow probably.

**Comment by Nik-Potokar** [comment]
1.7.7 Update is live with this feature.

Please test it and let us know if it needs any changes!

Thank you for all the work!!!  @Codemned

**Comment by opengrip** [comment]
Tested and seems to be working perfect :)
